Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -623,11 +637,154 @@ - (void)fetchProfileForRecipientId:(NSString *)recipientId
     }
 
     userProfile.lastUpdateDate = [NSDate new];
-    [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
-        [userProfile saveWithTransaction:transaction];
-    }];
 
-    // TODO: Actually update the profile.
+    [self saveUserProfile:userProfile];
+
+    [ProfileFetcherJob runWithRecipientId:recipientId
+                           networkManager:self.networkManager
+                         ignoreThrottling:ignoreThrottling];
```

Comment:
What I mean is ProfileFetcherJob has a throttling mechanism.
This method has a separate throttling mechanism.

This methods throttling mechanism disregards the ProfileFetcherJob's throttling mechanism.

I'm hopeful that there's an opportunity to consolidate the two throttling mechanisms and make the code easier to reason about.